src/main/rx/rx.c

```diff
@@ -105,6 +105,14 @@ static uint8_t rcSampleIndex = 0;
 #define SERIALRX_PROVIDER 0
 #endif
 
+#ifndef SERIALRX_INVERTED
+#if (SERIALRX_PROVIDER == SERIALRX_PROVIDER_SBUS)
+#define SERIALRX_INVERTED 1
```

Comment:
This should not be done. Since the default for inversion on SBus is `on`, setting `serialrx_inverted` to `on` as well means that inversion for SBus will be turned off, making it not work with most gear.

Reply:
Where is that inversion default set? I can only see serialrx_inverted = OFF. 
If that means inversion is ON and setting serialrx_inverted = ON means inversion is OFF, then something is really backwards. I do not follow this.... Do we have dual inversion? Inverted inversion??